src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -445,7 +445,8 @@ private static void InitializeGetEnumerableCache()
 
         internal static bool IsTypeEnumerable(Type type)
         {
-            if (type == null) { return false; }
+            if (type == null)
+            { return false; }
```

Comment:
```suggestion
            {
                return false;
            }
```

Reply:
Argh, autoformatter got me again. I'll do a style pass after this PR, but I'll take these out for now as they aren't functional changes.